ledger/txtail_test.go

```diff
@@ -58,7 +58,7 @@ func TestTxTailCheckdup(t *testing.T) {
 		txleases := make(map[ledgercore.Txlease]basics.Round)
 		txleases[ledgercore.Txlease{Sender: basics.Address(crypto.Hash([]byte{byte(rnd % 256), byte(rnd / 256), byte(2)})), Lease: crypto.Hash([]byte{byte(rnd % 256), byte(rnd / 256), byte(3)})}] = rnd + leasevalidity
 
-		tail.newBlock(blk, ledgercore.StateDelta{Accts: make(map[basics.Address]basics.AccountData), Hdr: &blk.BlockHeader, Txids: txids, Txleases: txleases})
+		tail.newBlock(blk, ledgercore.StateDelta{Accts: ledgercore.AccountsDelta{}, Hdr: &blk.BlockHeader, Txids: txids, Txleases: txleases})
```

Comment:
I'd follow your own pattern:  use MakeStateDelta(), and the follow up by updating the rest of the fields.

Reply:
done